src/components/Admin/AdminComponents/CreateListing.js

```diff
@@ -37,6 +37,7 @@ const CreateListing = () => {
     // Function that changes the state of the overlay when button is clicked
     const handleClose = () => {
         setOpen(false);
+        window.location.href = "http://localhost:3000/admin/listing";
```

Comment:
Please replace this with REACT_APP_FLASK_SERVER + "/admin/listing" 

This is important because when in deployment, the frontend is going to be calling the production URL and we control the URLs and Routes through netlify's environment variable.